packages/react-native-room-kit/src/components/HMSManageAudioOutput.tsx

```diff
@@ -120,23 +119,27 @@ export const HMSManageAudioOutput: React.FC = () => {
   }, [hmsInstance, debugMode]);
 
   // Handle changing selected audio device
-  const handleSelectAudioDevice = (device: HMSAudioDevice) => {
-    hmsInstance.switchAudioOutput(device);
+  const handleSelectAudioDevice = (device: HMSAudioDevice | 'mute-audio' | 'ios-audio-device') => {
```

Comment:
- Do not register clicks if the option is already selected
- Highlight the selected option